### DISCUSSION TITLE
Using Ctrl+v to paste images or video from clipboard to the playlist

### DISCUSSION THREAD

**Discussion by dagandrew** [sircharlo/meeting-media-manager#2764] [Ideas] [answered]
for example, you've copied the image from the browser or from a chat app (that doesn't support drag-and-drop) and you could just paste it in the playlist

**Comment by sircharlo** [comment]
I'll have to look into paste from clipboard; seems like it should technically be possible but I'll research it.

**Reply by sircharlo** [reply, marked as answer]
This feature isn't fully implemented in Electron yet (see [Electron Issue #26377](https://github.com/electron/electron/issues/26377)). Currently, some data types can be pasted (e.g., images), some are partially supported (e.g., a single file copied from the system file explorer), and others require OS-specific workarounds (e.g., multiple files copied from the system file explorer). We'll wait for an official enhancement to the Clipboard API unless this becomes a critical requirement.
